lib/rubocop-rspec.rb

```diff
@@ -37,7 +35,10 @@
 
```

Comment:
```
require_relative 'rubocop/rspec/factory_bot'
```
above is needed by both the extracted (RSpec-agnostic) and the remaining (RSpec-specific) FactoryBot cops.